x/marker/client/cli/cli_test.go

```diff
@@ -220,14 +220,46 @@ func (s *IntegrationTestSuite) TestMarkerTxCommands() {
 			markercli.GetCmdAddMarker(),
 			[]string{
 				"1000hotdog",
-				"--type=RESTRICTED",
+				fmt.Sprintf("--%s=%s", markercli.FlagType, "RESTRICTED"),
+				fmt.Sprintf("--%s=%s", markercli.FlagSupplyFixed, "true"),
+				fmt.Sprintf("--%s=%s", markercli.FlagSupplyFixed, "true"),
```

Comment:
I think this second one should be for governance control...

Reply:
🤦